README.md

```diff
@@ -1,24 +1,59 @@
-# 👾 Processbot
+# 👾 processbot
 
 # Commands
 
 To be posted as a comment in the Pull Request. **Your whole comment should only have the command**.
 
-- `bot merge` to automatically merge it once checks pass (if approvals have been
-  given).
-- `bot merge force` to attempt merge without waiting for checks (if approvals
-  have been given).
-- `bot merge cancel` to cancel a pending `bot merge`.
-- `bot compare substrate` to see a diff between current branch's Substrate.
+- `bot merge`: [if approved](#criteria-for-merge), merge once checks pass.
+- `bot merge force`: [if approved](#criteria-for-merge), merge immediately
+  while disregarding checks.
+- `bot merge cancel`: cancel a pending `bot merge`.
```

Comment:
What exactly does it do? Could you unwind the description. I'm interested i.e. if it sends the signal to the according gitlab pipeline to stop running.